pom.xml

```diff
@@ -209,8 +209,8 @@
         <version.weld>5.0.1.Final</version.weld>
         
         <version.wildfly.plugin>4.0.0.Final</version.wildfly.plugin>
-        <version.wildfly.core>18.1.2.Final</version.wildfly.core>
-        <version.wildfly.logging>5.0.0.Beta5</version.wildfly.logging>
+        <version.wildfly.core>20.0.2.Final</version.wildfly.core>
+        <version.wildfly.logging>5.0.0.Final</version.wildfly.logging>
```

Comment:
Could you please move these version bumps to another commit? Thanks

Reply:
Thanks @jmfinelli , sure!